### PR TITLE
Update 4

### PR DESCRIPTION
**Pull request by Rainb0wKey**
All "A"

### CODE REVIEW

**Comment by Rainb0wKey** [comment]
Tomorrow I plan to update all pages to "B"

File: src/bdscript/authorURL.md [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
 ```
-> 🧙‍♂️ `$authorURL` will not work if there is no text provided in `$author`.
+> `$authorURL[]` will not work if there is no text provided in `$author[]`.
 
-### Breakdown
-- `url` - The link to set as the author hyperlink.
-- `index` - What embed the author url should belong to (Optional). Default is `1`. [(learn more)](../resources/embedIndexes.md)
+### Parameters
+- `url`: The link to set as the author hyperlink.
+- `index`: What embed the author url should belong to. (Optional) [(learn more)](../resources/embedIndexes.md)
```
Comment (MineBartekSA) [inline src/bdscript/authorURL.md]:
```suggestion
- `index`: To which embed the author URL will be added. (Optional) [(learn more)](../resources/embedIndexes.md)
```

File: src/bdscript/authorOfMessage.md [hunk 2]
```diff
@@ -1,12 +1,22 @@
...
 
-### Breakdown
-- `channelID` - The channel where the message was sent.
-- `messageID` - The message that the bot returns the author ID for.
-> 🧙‍♂️ [Where do I find message/channel IDs? (click-me)](https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-)
+## Parameters
+- `channel ID`: The channel where the message was sent.
+- `message ID`: The message that the bot returns the author ID for.
+
+>  [How get message/channel ID guide.](https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-)
```
Comment (MineBartekSA) [inline src/bdscript/authorOfMessage.md]:
```suggestion
>  [How to get the message/channel ID guide.](https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-)
```

File: src/bdscript/authorOfMessage.md [hunk 3]
```diff
@@ -1,12 +1,22 @@
...
+$authorOfMessage[channel ID;message ID]
 ```
 
-### Breakdown
-- `channelID` - The channel where the message was sent.
-- `messageID` - The message that the bot returns the author ID for.
-> 🧙‍♂️ [Where do I find message/channel IDs? (click-me)](https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-)
+## Parameters
+- `channel ID`: The channel where the message was sent.
+- `message ID`: The message that the bot returns the author ID for.
```
Comment (MineBartekSA) [inline src/bdscript/authorOfMessage.md]:
```suggestion
- `channel ID`: The channel where the message was sent in.
- `message ID`: The message for which the author ID will be returned.
```

File: src/bdscript/authorIcon.md [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
 ```
-
-### Breakdown
-- `image url` - The image that appears in the author icon. This must be a valid image URL.
-- `index` - What embed the author icon should belong to (Optional). Default is `1`. [(learn more)](../resources/embedIndexes.md)
+> `$authorIcon[]` will not work if there is no text provided in `$author[]`.
+> 
+## Parameters
+- `image url`: The image that appears in the author icon. This must be a valid image URL.
+- `index`: What embed the author icon should belong to. _(Optional)_ [(learn more)](../resources/embedIndexes.md)
```
Comment (MineBartekSA) [inline src/bdscript/authorIcon.md]:
```suggestion
- `image url`: The image for the author icon. This must be a valid image URL.
- `index`: To which embed the author icon will be added. _(Optional)_ [(learn more)](../resources/embedIndexes.md)
```

File: src/bdscript/author.md [hunk 5]
```diff
@@ -1,18 +1,19 @@
...
 ```
 $author[text;(index)]
 ```
 
-### Breakdown
-- `text` - The text that appears in the author text. It cannot exceed more than 256 characters.
-- `index` - What embed the author text should belong to (Optional). Default is `1`. [(learn more)](../resources/embedIndexes.md)
+## Parameters
+- `text`: The text that appears in the author text. It cannot exceed more than 256 characters.
+- `index`: What embed the author text should belong to. _(Optional)_ [(learn more)](../resources/embedIndexes.md)
```
Comment (MineBartekSA) [inline src/bdscript/author.md]:
```suggestion
- `index`: To which embed the author text will be added. _(Optional)_ [(learn more)](../resources/embedIndexes.md)
```

**Review by MineBartekSA — CHANGES_REQUESTED** [review]

**Review by MineBartekSA — APPROVED** [review]
Thank you for your contribution

**Pull request merged**